Classes/Repository/RepositoryCodeBlobViewController.swift

```diff
@@ -100,11 +117,21 @@ final class RepositoryCodeBlobViewController: UIViewController {
     func handle(text: String) {
         emptyView.isHidden = true
 
-        textView.text = text
+        textView.attributedText = CreateColorCodedString(code: text, includeDiff: false, lineHandler: nil)
         let max = CGFloat.greatestFiniteMagnitude
         let size = textView.sizeThatFits(CGSize(width: max, height: max))
         textView.frame = CGRect(origin: .zero, size: size)
         scrollView.contentSize = size
     }
 
+    // MARK: GutterLayoutManagerDelegate implementation
+
+    func getMaximumGutterWidth(gutterLayoutManager: GutterLayoutManager) -> CGFloat {
```

Comment:
This looks identical to the implementation in GutterLayoutManager (ext. GutterLayoutManagerDelegate)